src/Microsoft.SqlTools.ServiceLayer/DacFx/ExtractOperation.cs

```diff
@@ -26,7 +26,7 @@ public ExtractOperation(ExtractParams parameters, ConnectionInfo connInfo) : bas
         public override void Execute()
         {
             Version version = ParseVersion(this.Parameters.ApplicationVersion);
-            DacExtractOptions extractOptions = GetExtractOptions(this.Parameters.ExtractTarget);
+            DacExtractOptions extractOptions = GetExtractOptions(this.Parameters.ExtractTarget, this.Parameters.IncludePermissions);
```

Comment:
Are there any tests for this where we should update it?

Reply:
since it's an optional parameter, the existing tests aren't affected and they're passing in the Integration tests PR check :)